includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -0,0 +1,285 @@
+		'default' => [
+			'fandom-logo' => 'http://fandom.wikia.com',
+			'games' => 'http://fandom.wikia.com/games',
+			'movies' => 'http://fandom.wikia.com/movies',
+			'tv' => 'http://fandom.wikia.com/tv',
+			'explore-wikis' => 'http://fandom.wikia.com/explore',
+			'community-central' => 'http://community.wikia.com/wiki/Community_Central',
+			'fandom-university' => 'http://community.wikia.com/wiki/Wikia_University',
+			'user-signin' => 'https://www.wikia.com/signin',
+			'user-register' => 'https://www.wikia.com/register',
+		],
+		'en' => [ ]
+	];
+
+	private $wikiId;
+	private $lang;
+
+	public function __construct( $wikiId, $lang = self::DEFAULT_LANG ) {
+		parent::__construct();
+
+		$this->wikiId = $wikiId;
+		$this->lang = $lang;
+	}
+
+	public function getData() {
+		global $wgUser;
+
+		$data = [
+			'logo' => [
+				'header' => [
+					'type' => 'link-image',
+					'href' => $this->getHref( 'fandom-logo' ),
+					'image' => 'wds-company-logo-fandom',
+					'title' => [
+						'type' => 'text',
+						'value' => 'Fandom powered by Wikia'
+					]
+				]
+			],
+			'verticals' => [
+				'links' => [
+					[
+						'type' => 'link-branded',
+						'brand' => 'tv',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-brandlink-vertical-tv'
+						],
+						'href' => $this->getHref( 'tv' ),
+					],
+					[
+						'type' => 'link-branded',
+						'brand' => 'games',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-brandlink-vertical-games'
+						],
+						'href' => $this->getHref( 'games' ),
+					],
+					[
+						'type' => 'link-branded',
+						'brand' => 'movies',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-brandlink-vertical-movies'
+						],
+						'href' => $this->getHref( 'movies' ),
+					]
+				]
+			],
+			'wikis' => [
+				'header' => [
+					'type' => 'line-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-wikis',
+					],
+				],
+				'links' => [
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-brandlink-wikis-explore'
+						],
+						'href' => $this->getHref( 'explore-wikis' ),
+					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-wikis-communitycentral'
+						],
+						'href' => $this->getHref( 'community-central' ),
+					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-wikis-fandomuniversity'
+						],
+						'href' => $this->getHref( 'fandom-university' ),
+					]
+				]
+			],
+			'search' => [
+				'module' => [
+					'type' => 'search',
+					'results' => [
+						'url' => $this->getPageUrl( 'Search', NS_SPECIAL, 'fulltext=Search' ),
+						'param-name' => 'query'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "param-name" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)